src/core_modules/capture-core/components/WorkingLists/EventWorkingLists/ReduxProvider/EventWorkingListsReduxProvider.container.js

```diff
@@ -102,7 +99,7 @@ export const EventWorkingListsReduxProvider = ({ storeId, program, programStage,
             onLoadView={injectDownloadRequestToLoadView}
             onUpdateList={injectDownloadRequestToUpdateList}
             onDeleteEvent={onDeleteEvent}
-            downloadRequest={downloadRequest}
+            storeId={storeId}
```

Comment:
Can we keep the downloadRequest in the props list instead of the storeId? I know you will have to use UseSelector twice (once in the events list and once in the TE-list), but I will be easier for us going forward I think.

Reply:
Sure, I implemented your suggestions in the last commit. Thanks for the feedback!